malaria24/ona/tests/test_tasks.py

```diff
@@ -36,6 +41,9 @@ def setUp(self):
                       body=pkg_resources.resource_string(
                           'malaria24', 'ona/fixtures/responses/forms.json'))
         post_save.disconnect(new_case_alert_ehps, sender=ReportedCase)
+        self.complete_url = FAKE_URL
```

Comment:
These won't be needed as you will just get them from settings where you use them.